app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,73 @@
+<p style="color: green"><%= notice %></p>
+
+<section>
+  <h2 class='recipe-name'><%= @recipe.name %></h2>
+
+  <div class='recipe-main'>
+    
+    <div class='public-btn-div'>
+      <p class='sub-title'>Preparation time : <%= @recipe.preparation_time %></p>
+      <div class='public-btn-flex'>
+        <p class='sub-title'>Public :</p>
+        <%= button_to  recipe_path(@recipe.id), :method => :patch do %>
+          <% if @recipe.public %>
+          <% end %>
+          <% if !@recipe.public %>
+        
+          <% end %>
+          <% end %>
+      </div>
+    </div>
+    <div>
+    <p class='sub-title'>Cooking time : <%= @recipe.cooking_time %></p>
+    </div>
+
+    <div>
+    <p class='sub-title'>Steps goes here ...</p>
+    </div>
+
+    <div class='bottom-butoons'>
+    <%= link_to 'General Shopping list', general_shopping_list_index_path(@food_recipe.recipe.id), class: "shopping-link" %>
+    <%= link_to 'Add ingredient', edit_food_recipe_path(@food_recipe.recipe.id), class: "ingredient-link"%>
```

Comment:
- [ ] When I try to add an ingredient to a Recipe, I'm getting this error:

![image](https://github.com/gsmalik030/recipe_app/assets/77069011/89496158-6766-41b2-ad91-8c152fd33b61)

Please consider taking a look at it and fixing it, because, without this functionality, I'm unable to review your `General Shopping List`. 🤓 🔧